src/main/java/gift/model/Wish.java

```diff
@@ -9,33 +9,65 @@ public class Wish {
     private Long id;
 
     @ManyToOne
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.Lazy)
```
로 바꾸는 걸 추천드립니다. 지연 로딩의 개념은 아마 Fetch Join 개념에서 제일 중요한 거라 찾아보시면 나올거에요!